src/util/string-helper.ts

```diff
@@ -1,3 +1,5 @@
+import { MaskAttributeProps } from "../masked-input";
```

Comment:
hmm feels odd to have a util import from a component, even if it's just typing. perhaps this component could expose it as an option instead? and let the components re-export as needed?

Reply:
I found it awkward also but worried we would replicate. I think maybe it's more direct if i respecify the option types in the helper instead.